augur/tasks/start_tasks.py

```diff
@@ -372,7 +372,7 @@ def augur_collection_monitor():
 
         if facade_phase.__name__ in enabled_phase_names:
             #Schedule facade collection before clone/updates as that is a higher priority
-            start_facade_collection(session, max_repo=30, days=30)
```

Comment:
I'm slightly worried that this might crowd out the other task processes since facade shares a worker queue with the core task collection. Now that we are switching to having 50 processes available for the core worker we could do something like splitting it 50/50 by having start_tasks schedule 25 repos each for facade and core collection.

Reply:
@IsaacMilarky : OK. This makes a lot of sense. I did not understand that this was coming out of the core allocation. I will adjust.

Reply:
Shouldn't we make the primary collection start 30 repos and then have facade collection start 30 repos to evenly distribute the available processes among the repos being scheduled since we have 60 processes? I would prefer to not put more messages on the queue than we need to and in this pr it is scheduling 85 repos at once for 60 processes. Especially since we aren't using eventlet anymore. Is there a reason we are scheduling so many repos at once?  We will still have to keep the messages for these repos in the queue bc of all the jobs we are doing for each repo but under this configuration we have like ~25 repos just sitting in the rabbitmq queue doing nothing and not even have a reason to be in the queue because none of those would have tasks being processed by the worker processes at that point.